modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -484,11 +485,7 @@ object SeqexecEngine {
 
   // scalastyle:off
   def seqexecConfiguration(giapi: Giapi[IO]): Kleisli[IO, Config, Settings] = Kleisli { cfg: Config =>
-    val site = cfg.require[String]("seqexec-engine.site") match {
-      case "GS" => Site.GS
-      case "GN" => Site.GN
-      case _    => Site.GS // Let's default to GS
-    }
+    val site                    = cfg.require[Site]("seqexec-engine.site")
```

Comment:
@tpolecat This looks much better. I guess I'd like not to write these as String but `knobs` doesn't let me override that. Anyway this is less boilerplaty and much more explicit